package.json

```
@@ -53,7 +53,7 @@
     "jest": "jest",
     "license-check": "jsgl --local ./",
     "lint": "yarn eslint && yarn lint-check && yarn pretty-check",
-    "lint-check": "eslint --print-config .eslintrc.js | eslint-config-prettier-check",
+    "lint-check": "eslint --print-config .eslintrc.js",
```

Looking at the link you posted, shouldn't we now do this to check for conflicting rules? The current script only prints the config.
```suggestion
    "lint-check": "eslint-config-prettier .eslintrc.js",
```